amplipi/updater/static/index.html

```diff
@@ -105,6 +110,71 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
                 </div>
               </div>
             </div>
+
+            <div class="tab-pane fade text-left" id="admin-settings" role="tabpanel" aria-labelledby="admin-settings-tab">
+              <script>
+                async function getPersist() {
+                  const response = await fetch("/settings/persist_logs", { method: "GET" });
+                  const data = await response.json();
+                  const checkbox = document.getElementById("persist_logs_checkbox");
+                  const textbox = document.getElementById("persist_logs_delay_input");
+                  checkbox.checked = data.persist_logs;
+                  textbox.value = data.auto_off_delay;
+                }
+
+                async function setPersist() {
+                  const checkbox = document.getElementById("persist_logs_checkbox");
+                  const textbox = document.getElementById("persist_logs_delay_input");
+                  const response = await fetch("/settings/persist_logs", {
+                    method: "POST",
+                    headers: {
+                        "Content-Type": "application/json",
+                    },
+                    body: JSON.stringify({
+                        persist_logs: checkbox.checked,
+                        auto_off_delay: parseInt(textbox.value),
+                    }),
+                  });
+                }
+
+                window.onload = getPersist;
+              </script>
+
+              <div id="admin-settings-dialog">
+                This tab allows you to change settings relating to the core of AmpliPi
+              </div>
+              <p class="">
+                <a class="btn btn-primary btn-sm text-center" id="set-password-button" data-toggle="tab" href="#set-password" role="tab" aria-controls="set-password" aria-selected="false">Set Password</a>
+              </p>
+
+              <!-- Log Persistence -->
+              <div id="set-password" style="display: flex; flex-direction: column; gap: 10px;">
+                <div class="set-password form-group">
+                  <h6>
+                    Log Persistence
+                  </h6>
+                  <div class="set-password-dialog">
+                    WARNING: leaving log persistence on for extended periods can damage your system.
+                    <br/>
+                    As such, we suggest also using the "Auto off" function, that will deactivate persist logs after the input number of days.
```

Comment:
```suggestion
                    As such, we suggest also using the "Auto off" function. This will deactivate persistant logs after the specified number of days.
```